CONTRIBUTING.md

```diff
@@ -129,6 +129,44 @@ make html
 
 The built docs should now be available in `docs/build/html`
 
+## Conventions
+
+With torchaudio being a machine learning library and built on top of PyTorch,
+torchaudio is standardized around the following naming conventions. Tensors are
+assumed to have "channel" as the first dimension and time as the last
+dimension (when applicable). This makes it consistent with PyTorch's dimensions.
+For size names, the prefix `n_` is used (e.g. "a tensor of size (`n_freq`, `n_mels`)")
+whereas dimension names do not have this prefix (e.g. "a tensor of
+dimension (channel, time)")
+
+* `waveform`: a tensor of audio samples with dimensions (channel, time)
+* `sample_rate`: the rate of audio dimensions (samples per second)
+* `specgram`: a tensor of spectrogram with dimensions (channel, freq, time)
+* `mel_specgram`: a mel spectrogram with dimensions (channel, mel, time)
+* `hop_length`: the number of samples between the starts of consecutive frames
+* `n_fft`: the number of Fourier bins
+* `n_mels`, `n_mfcc`: the number of mel and MFCC bins
+* `n_freq`: the number of bins in a linear spectrogram
+* `f_min`: the lowest frequency of the lowest band in a spectrogram
+* `f_max`: the highest frequency of the highest band in a spectrogram
+* `win_length`: the length of the STFT window
+* `window_fn`: for functions that creates windows e.g. `torch.hann_window`
+
+Transforms expect and return the following dimensions.
```

Comment:
Let's remove the transforms. This is not a convention. The same thing is explained in the doc, and there should be only one source of truth. Also this is not necessarily relevant to new contributors.

Reply:
I agree with this. If I need to know the input or output, I would just go to the documentation website instead of here.